src/raylib.h

```diff
@@ -1425,6 +1425,7 @@ RLAPI int ColorToInt(Color color);                                          // G
 RLAPI Vector4 ColorNormalize(Color color);                                  // Get Color normalized as float [0..1]
 RLAPI Color ColorFromNormalized(Vector4 normalized);                        // Get Color from normalized values [0..1]
 RLAPI Vector3 ColorToHSV(Color color);                                      // Get HSV values for a Color, hue [0..360], saturation/value [0..1]
+RLAPI Color ColorLerp(Color color1, Color color2, float d);                 // Mix 2 Colors together   
```

Comment:
Please, could you move this function to the last position of Color() functions? Also, review the comment to follow consistency with other functions.